types/linkifyjs/index.d.ts

```diff
@@ -13,7 +13,11 @@ export type PossiblyFuncOfHrefAndType<T> =
 
 // There's always a possibility that values at a string key can always be
 // undefined; i.e., if the key does not exist.
-export type PossiblyByType<T> = T | { [type: string]: T | undefined };
+export type PossiblyByType<T> = T | { [type: string]: T | React.ReactElement | undefined };
```

Comment:
What was this modification for? `PossiblyByType` is sort of a type that deals mostly with logic. I don't think `React.ReactElement` belongs here.

If there's a specific reason you were trying to use `React.ReactElement` for but you're not sure how to do it, feel free to explain and I can see if I can come up with a way to solve it as well.